src/python/chrome/build_info.py

```diff
@@ -75,14 +86,48 @@ def get_production_builds_info(platform):
   return builds_metadata
 
 
+def get_production_builds_info_from_cd(platform):
+  """Gets the build information from Chromium Dash for production builds.
+
+  Omits platforms containing digits, namely, win64.
+  Omits channels containing underscore, namely, canary_asan.
+  Platform is e.g. ANDROID, LINUX, MAC, WIN.
+  """
+  builds_metadata = []
+  chromiumdash_platform = _convert_platform_to_chromiumdash_platform(platform)
+  query_url = BUILD_INFO_URL_CD % chromiumdash_platform
+
+  build_info = utils.fetch_url(query_url)
+  if not build_info:
+    logs.log_error('Failed to fetch build info from %s' % query_url)
+    return []
+
+  try:
+    build_info_json = json.loads(build_info)
+    if len(build_info_json) == 0:
+      logs.log_error('Empty response from %s' % query_url)
+      return []
+  except Exception as e:
+    logs.log_error('Malformed response from %s: %s' % (query_url, str(e)))
```

Comment:
No need to include the exception in the string. Our log_error will automatically include it.